### ISSUE TITLE
Poor color contrast on files bars, inconsistency with assignment bars

### ISSUE THREAD

**Issue by jonespm** [tl-its-umich-edu/my-learning-analytics#748]
## Expected behavior (A clear and concise description of what you expected to happen) : 

On the files page, the bars have a poor color contrast making the number very difficult to read. 
The background is #808080 and text is #0F0F0F, which has a 4.5:1 ratio, barely passing.

Files:
![image](https://user-images.githubusercontent.com/27447/66310139-6e71a680-e8d9-11e9-8429-fb3ffabe025f.png)

Assignment has a background of #E1E1E1 which has a 14:1 ratio.
![image](https://user-images.githubusercontent.com/27447/66310156-7af5ff00-e8d9-11e9-923f-19ead0fac54f.png)

**Comment by ssciolla** [comment]
This is related to issue #673, which I'm working on and hope to have a pull request soon.

**Comment by jennlove-um** [comment]
The contrast between the gray bar and the black text is now 10:1 using #B5B5B5 for the gray.

**Comment by jennlove-um** [comment]
Testing passes.

**Comment by gsilver** [comment]
@pushyamig @ssciolla - nice.
